src/components/formSections/components/TermsListField.js

```diff
@@ -259,28 +328,46 @@ export default class TermsListField extends React.Component {
     let termNoteError = false;
 
     const termsList = terms.map((term, i) => {
-      const errorMessage = this.validateNoteField(value, term.value);
+      const errorMessage = this.validateFields(value, term.value);
       termNoteError = errorMessage ? true : termNoteError;
 
       return (
-        <React.Fragment key={term.value}>
+        <Card
```

Comment:
Could we be using a stripes-erm-components `EditCard` for this instead? It'd allow you to not render the `headerEnd`. See `DocumentsFieldArray` for an example of usage.

Reply:
i have a problem with the onDeleteField property, i don't really know where it comes from. the trash bin isn't functional yet